src/index.ts

```diff
@@ -348,12 +351,115 @@ async function certificateForImpl<
     key: readFile(domainKeyPath),
     cert: readFile(domainCertPath)
   } as IReturnData<O>;
-  if (options.getCaBuffer) (ret as CaBuffer).ca = readFile(rootCACertPath);
-  if (options.getCaPath) (ret as CaPath).caPath = rootCACertPath;
+  if (options.getCaBuffer)
+    ((ret as unknown) as CaBuffer).ca = readFile(rootCACertPath);
+  if (options.getCaPath) ((ret as unknown) as CaPath).caPath = rootCACertPath;
 
   return ret;
 }
 
+/**
+ * Trust the certificate for a given hostname and port and add
+ * the returned cert to the local trust store.
+ * @param hostname - hostname of the remote machine
+ * @param port - port to connect the remote machine
+ * @param certPath - file path to store the cert
+ */
+async function trustCertsOnRemote(
+  hostname: string,
+  port: number,
+  certPath: string
+): Promise<string> {
+  // Get the remote certificate from the server
+  const certData = await getRemoteCertificate(hostname, port);
+  try {
+    // Write the certificate data on this file.
+    writeFileSync(certPath, certData);
+  } catch (err) {
+    throw new Error(err);
+  }
+  // Trust the remote cert on your local box
+  try {
+    await currentPlatform.addToTrustStores(certPath);
+    debug('Certificate trusted successfully');
+    debug('Attempting to close the remote server');
+  } catch (err) {
+    throw new Error(err);
+  }
+  return certData;
+}
+/**
+ * Trust the remote hosts's certificate on local machine.
+ * This function would ssh into the remote host, get the certificate
+ * and trust the local machine from where this function is getting called from.
+ * @public
+ * @param hostname - hostname of the remote machine
+ * @param port - port to connect the remote machine
+ * @param certPath - file path to store the cert
+ * @param renewalBufferInBusinessDays - valid days before renewing the cert
+ */
+export function trustRemoteMachine(
+  hostname: string,
+  port: number,
+  certPath: string,
+  renewalBufferInBusinessDays = REMAINING_BUSINESS_DAYS_VALIDITY_BEFORE_RENEW
+): Promise<boolean> {
+  return new Promise((resolve, reject) => {
+    debug(`Connecting to remote server on port: ${port}`);
+    // Connect to remote box via ssh.
+    const child = execa.shell(
+      // @TODO Change this to npx
+      `ssh ${hostname} node devcert/src/express.js --port=${port} `,
```

Comment:
Isn't this assuming that `devcert/src/express.js` exists as a subdirectory of the user's `$HOME`? Is that guaranteed to be true?